Makefile

```diff
@@ -70,6 +70,7 @@ e2e-test:
 
 .PHONY: e2e-bootstrap
 e2e-bootstrap: install-helm
+ifndef E2E_SKIP_INSTALL
```

Comment:
so in this PR, when run `make e2e-bootstrap`, it only installs helm, why not user don't run `make e2e-bootstrap`

Reply:
maybe we don't need this `E2E_SKIP_INSTALL` flag, and only run `make e2e-test` could solve the issue?

Reply:
It's part of the e2e test suite - https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/test/e2e/suite_test.go#L88-L93. We also shouldn't install helm. I can modify the PR.